ui/components/multichain/network-list-item/network-list-item.stories.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { NetworkListItem } from '.';
+
+export default {
+  title: 'Components/Multichain/NetworkListItem',
+  component: NetworkListItem,
+  argTypes: {
+    name: {
+      control: 'text',
+    },
+    selected: {
+      control: 'boolean',
+    },
+    onClick: {
+      action: 'onClick',
+    },
+    onDeleteClick: {
+      action: 'onClick',
+    },
```

Comment:
action should be prop name
```suggestion
    onDeleteClick: {
      action: 'onDeleteClick',
    },
```